web/src/Web.App/Views/DataSource/_DataSources.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿<h2 class="govuk-heading-m">Data Sources</h2>
+
+<h3 class="govuk-heading-s">Which types of schools can I get data for?</h3>
+
+<p class="govuk-body">
+    The data on the site is for maintained schools and academies in England. It’s based on schools’ Consistent Financial Reporting (CFR) returns and academies’ accounting returns.
+    You can also get data for multi-academy trusts (MATs).
+</p>
+
+<h3 class="govuk-heading-s">Financial returns</h3>
+
+<p class="govuk-body">
+    This is based on schools’ Consistent Financial Reporting (CFR) returns for LA maintained schools and academies’ accounting returns (AARs).
+</p>
+
+<h4 class="govuk-heading-s">LA Maintained Schools</h4>
+
+<ul class="govuk-list">
+    @foreach (var dataSourceFile in Model.MaintainedSchools)
+    {
+        <li>
+            <a target="_blank" rel="external noopener noreferrer" class="govuk-link" href=@dataSourceFile.Link>
```

Comment:
suggestion: This link will reveal the location in blob storage, including the SAS Token. An alternative consideration could be an API action that proxies this request instead

Reply:
I'm happy with this approach. 

It's a separate storage account, and the sas is time bound & limited access.